script.js

```diff
@@ -0,0 +1,32 @@
+let basic_monthly = document.getElementById("basic_monthly");
+let basic_annualy = document.getElementById("basic_annualy");
+
+let professioanal_monthly = document.getElementById("professioanal_monthly");
+let professioanal_annualy = document.getElementById("professioanal_annualy");
+
+let master_monthly = document.getElementById("master_monthly");
+let master_annualy = document.getElementById("master_annualy");
+
+let toogle = document.getElementById("toogle_button");
+
```

Comment:
If you are not updating your variables, use `const` not `let` 
Update it here and everywhere else necessary